src/planscape/rscripts/forsys.R

```diff
@@ -167,97 +267,109 @@ get_configuration <- function(scenario) {
   return(configuration)
 }
 
-call_forsys <- function(connection, scenario) {
-  now <- now_utc()
-  configuration <- get_configuration(scenario)
-  conditions <- get_conditions(connection, scenario, configuration)
-  stand_data <- get_stand_data(connection, scenario, conditions)
+call_forsys <- function(
+    connection,
+    scenario,
+    configuration,
+    priorities,
+    outputs) {
+  forsys_inputs <- rbind(priorities, outputs)
+  stand_data <- get_stand_data(connection, scenario, forsys_inputs)
 
-  if (length(names(conditions)) > 1) {
+  if (length(priorities$condition_name) > 1) {
     stand_data <- stand_data %>% forsys::combine_priorities(
-      fields = names(conditions),
+      fields = priorities$condition_name,
       weights = configuration$weights,
       new_field = "priority"
     )
     scenario_priorities <- c("priority")
   } else {
-    scenario_priorities <- names(conditions)
+    scenario_priorities <- first(priorities$condition_name)
   }
-
+  number_of_projects <- 10
```

Comment:
can you leave a comment at the top of this assumption and other things we might allow users to specify in the future?  (this is a nonblocking request.)